### PR TITLE
Update Paolas coordinates

### PR DESCRIPTION
**Pull request by Paola191**
I want to add my pin to the map

### CODE REVIEW

File: _pins/Paola191.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+githubHandle: Paola191
+latitude: 59.3333
+longitude: 18.05
```
Comment (annaawgren) [inline _pins/Paola191.json]: Nice work on the longitude!👌🏻⚡️

File: _pins/Paola191.json [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+githubHandle: Paola191
+latitude: 59.3333
```
Comment (annaawgren) [inline _pins/Paola191.json]: Awesome latitude numbers!💯

File: _pins/Paola191.json (hunk 1 above)
Comment (Paola191) [reply _pins/Paola191.json]: Gracias Anna!

File: _pins/Paola191.json (hunk 2 above)
Comment (Paola191) [reply _pins/Paola191.json]: Tack Anna!

**Pull request merged**